src/main/java/io/stargate/sgv2/jsonapi/service/resolver/model/impl/FindOneAndUpdateCommandResolver.java

```diff
@@ -67,6 +67,14 @@ public Operation resolveCommand(CommandContext commandContext, FindOneAndUpdateC
   private FindOperation getFindOperation(
       CommandContext commandContext, FindOneAndUpdateCommand command) {
     List<DBFilterBase> filters = resolve(commandContext, command);
-    return new FindOperation(commandContext, filters, null, 1, 1, ReadType.DOCUMENT, objectMapper);
+    return new FindOperation(
+        commandContext,
+        filters,
+        command.buildProjector(),
```

Comment:
Not sure this is correct, the FindOperation inside FindOneAndUpdate should be passed identityProjector() because the document is merged with updates. Need to apply projection on top document that is returned [here] (https://github.com/stargate/jsonapi/blob/b6d4f73be517cf2e90fb50384e8e60f411754f99/src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/ReadAndUpdateOperation.java#L183).  This needs Projection passed into ReadAndUpdateOperation

Reply:
Makes sense, will change.

Reply:
Changed in 2 commits.